PrivateHospitals/backend/PrivateHospitals/PrivateHospitals.Core/Models/Users/AppUser.cs

```diff
@@ -4,6 +4,7 @@ namespace PrivateHospitals.Core.Models.Users;
 
 public class AppUser : IdentityUser
 {
+    public string? ExternalId { get; set; }
```

Comment:
This is not flexible approach. because if we Consume a few more companies, ids may still clash. Add CompanyId or SourceId to provide uniqueness